### PR TITLE
Addressing issues from Bump to Elixir v 1.11 #897

### PR DESCRIPTION
**Pull request by JSteigner**
connects to #835

#### Reviewer don't-forgets:

- [ ] Test coverage feels appropriate, given potential risk
- [ ] We're not doubling down on already-bad code
- [ ] If there are web UI changes, they don't add anything that could be considered client-side page navigation (unless pre-approved as being necessary by another engineer)
- [ ] If there are web UI changes, they don't add any AJAX form submits (unless pre-approved as being necessary by another engineer)
- [ ] If there are any lint rules disabled, they are disabled per-line, and were (in the reviewer's judgment) appropriate to disable
- [ ] Any new environment variables used in the app are both documented and have been added to both staging and production environments already
- [ ] Potential race conditions are either inconsequential, or the code prevents them from occurring.
- [ ] If this is a UI change that called for screenshots/GIFs, in the reviewer's judgement, they were included

### CODE REVIEW

File: lib/app_template_web/endpoint.ex [hunk 1]
```diff
@@ -37,6 +37,9 @@ defmodule AppTemplateWeb.Endpoint do
     plug(Phoenix.CodeReloader)
   end
 
+  plug Phoenix.LiveDashboard.RequestLogger,
+    param_key: "request_logger",
+    cookie_key: "request_logger"
```
Comment (lukeledet) [inline lib/app_template_web/endpoint.ex:42]: Nitpick: put a newline after this

File: lib/app_template_web/telemetry.ex [hunk 2]
```diff
@@ -0,0 +1,50 @@
+defmodule AppTemplate.Telemetry do
+  use Supervisor
+
+  import Telemetry.Metrics
+
+  @moduledoc """
```
Comment (lukeledet) [inline lib/app_template_web/telemetry.ex:6]: @moduledoc should be the first thing in the module

**Review by lukeledet — APPROVED** [review]
Some very small nits but looks good!

File: lib/app_template_web/endpoint.ex (hunk 1 above)
Comment (JSteigner) [reply lib/app_template_web/endpoint.ex:42]: ahh I was wondering about that, I'll add that in

File: lib/app_template_web/telemetry.ex (hunk 2 above)
Comment (JSteigner) [reply lib/app_template_web/telemetry.ex:6]: I was wondering about this too! Thanks for clarifying!

**Comment by JSteigner** [comment]
Awesome thanks guys! Even though this was easy stuff it was still pretty exhilarating to get my feet wet! I just pushed up Lukes changes. I believe you can close up issues  Set up Live Dashboard with telemetry #897 and Bump Elixir to v 1.11 #835 
after this?

**Pull request merged**